<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Concerto</class>
 <widget class="QMainWindow" name="Concerto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Concerto - Mozart</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image-Resources.qrc">
    <normaloff>:/icons/images/ucicon.png</normaloff>:/icons/images/ucicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="background">
       <brush brushstyle="Dense1Pattern">
        <color alpha="255">
         <red>218</red>
         <green>217</green>
         <blue>217</blue>
        </color>
       </brush>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="actionP_roperties"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuAs_sociation">
     <property name="title">
      <string>As&amp;sociation</string>
     </property>
     <property name="icon">
      <iconset resource="Image-Resources.qrc">
       <normaloff>:/icons/images/items/connector.png</normaloff>:/icons/images/items/connector.png</iconset>
     </property>
     <addaction name="action_Unidirectional"/>
     <addaction name="action_Bidirectional"/>
    </widget>
    <widget class="QMenu" name="menu_Geometry">
     <property name="title">
      <string>&amp;Geometry</string>
     </property>
     <property name="icon">
      <iconset resource="Image-Resources.qrc">
       <normaloff>:/icons/images/items/text.png</normaloff>:/icons/images/items/text.png</iconset>
     </property>
     <addaction name="action_Square"/>
     <addaction name="action_Rectangle"/>
     <addaction name="action_Circle"/>
     <addaction name="action_Triangle"/>
    </widget>
    <addaction name="action_Actor"/>
    <addaction name="action_Use_Case"/>
    <addaction name="separator"/>
    <addaction name="actionSub_Flow"/>
    <addaction name="actionAl_ternate_Flow"/>
    <addaction name="actionSe_curity_Flow"/>
    <addaction name="separator"/>
    <addaction name="menuAs_sociation"/>
    <addaction name="actionE_xtension_Point"/>
    <addaction name="actionUses"/>
    <addaction name="separator"/>
    <addaction name="menu_Geometry"/>
   </widget>
   <widget class="QMenu" name="menu_Object">
    <property name="title">
     <string>&amp;Object</string>
    </property>
    <addaction name="actionSend_Backward"/>
    <addaction name="actionSend_Forward"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Contents"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Close_Window"/>
    <addaction name="actionClose_Asll"/>
    <addaction name="separator"/>
    <addaction name="action_Tile"/>
    <addaction name="actionCa_scade"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionS_witch_Layout_Direction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Object"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/document-new.svg</normaloff>:/icons/images/menu/File/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>News</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/document-open.svg</normaloff>:/icons/images/menu/File/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/window-close.svg</normaloff>:/icons/images/menu/File/window-close.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/document-save.svg</normaloff>:/icons/images/menu/File/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/document-save-as.svg</normaloff>:/icons/images/menu/File/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/document-print.svg</normaloff>:/icons/images/menu/File/document-print.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionP_roperties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/document-properties.svg</normaloff>:/icons/images/menu/File/document-properties.svg</iconset>
   </property>
   <property name="text">
    <string>P&amp;roperties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/File/exit.svg</normaloff>:/icons/images/menu/File/exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/Edit/undo.svg</normaloff>:/icons/images/menu/Edit/undo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/Edit/redo.svg</normaloff>:/icons/images/menu/Edit/redo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="action_Actor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/items/user-add.png</normaloff>:/icons/images/items/user-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Actor</string>
   </property>
  </action>
  <action name="action_Use_Case">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/items/usecase-add.png</normaloff>:/icons/images/items/usecase-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Use Case</string>
   </property>
  </action>
  <action name="actionSub_Flow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/items/subflow.png</normaloff>:/icons/images/items/subflow.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;ub Flow</string>
   </property>
  </action>
  <action name="actionAl_ternate_Flow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/items/security.svg</normaloff>:/icons/images/items/security.svg</iconset>
   </property>
   <property name="text">
    <string>Al&amp;ternate Flow</string>
   </property>
  </action>
  <action name="actionSe_curity_Flow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/items/security.png</normaloff>:/icons/images/items/security.png</iconset>
   </property>
   <property name="text">
    <string>Se&amp;curity Flow</string>
   </property>
  </action>
  <action name="action_Unidirectional">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Unidirectional</string>
   </property>
  </action>
  <action name="action_Bidirectional">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Bidirectional</string>
   </property>
  </action>
  <action name="actionE_xtension_Point">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/items/extension.png</normaloff>:/icons/images/items/extension.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xtension Point</string>
   </property>
  </action>
  <action name="actionUses">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/items/uses.png</normaloff>:/icons/images/items/uses.png</iconset>
   </property>
   <property name="text">
    <string>Uses</string>
   </property>
  </action>
  <action name="action_Square">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Square</string>
   </property>
  </action>
  <action name="action_Rectangle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Rectangle</string>
   </property>
  </action>
  <action name="action_Circle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Circle</string>
   </property>
  </action>
  <action name="action_Triangle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Triangle</string>
   </property>
  </action>
  <action name="actionSend_Backward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Send &amp;Backward</string>
   </property>
  </action>
  <action name="actionSend_Forward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Send &amp;Forward</string>
   </property>
  </action>
  <action name="action_Contents">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/Help/help.svg</normaloff>:/icons/images/menu/Help/help.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Image-Resources.qrc">
     <normaloff>:/icons/images/menu/Help/info.svg</normaloff>:/icons/images/menu/Help/info.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Close_Window">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionClose_Asll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="actionCa_scade">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ca&amp;scade</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
  </action>
  <action name="actionS_witch_Layout_Direction">
   <property name="text">
    <string>S&amp;witch Layout Direction</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Image-Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>Concerto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Previous</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activatePreviousSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Next</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>activateNextSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Tile</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCa_scade</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Close_Window</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeActiveSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose_Asll</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeAllSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
